modules/olmv1-updating-an-operator.adoc

```diff
@@ -159,6 +161,53 @@ spec:
 ----
 <1> Update the version from `1.11.1` to `1.12.1`
 
+** If you want to define a range of acceptable update versions, edit your CR similar to the following example:
++
+.Example CR with a version range specified
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  version: >1.11.1, <1.13 <1>
+----
+<1> Specifies that the desired version range is greater than version `1.11.1` and less than `1.13`. For more information, see "Support for version ranges".
```

Comment:
> Specifies that the desired version range is greater than version `1.11.1` and less than `1.13`. For more information, see "Support for version ranges".

Should we also mention the "Version Comparison string" section? as:
> For more information, see "Support for version ranges" and "Version Comparison string".

BTW, can these help texts include hyperlinks to those actual sections?
- #olmv1-version-range-support_olmv1-installing-operator
- #olmv1-version-range-comparisons_olmv1-installing-operator

Reply:
Unfortunately, there are limitations in our docs tooling that keep us from having direct internal links in the modules.

Reply:
I see, let's just point out those two sections without the links then, thanks!

Reply:
Addressed in https://github.com/openshift/openshift-docs/pull/71400/commits/1a454da70484267a60c68b64dde7d8fcca0d2c08.